aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -1,49 +1,71 @@
 # Marketing App Resources
 # This template is manually deployed to create/update the Marketing App's AWS resources.
 
+<%
+site = Config::MARKETING_SITES[site_short_code.to_sym]
+BASE_DOMAIN_NAME = Config::STAGES[stage.to_sym][:baseDomain]
+
+LOAD_BALANCER_DOMAIN_NAME = "#{site[:subdomain]}-loadbalancer.#{BASE_DOMAIN_NAME}"
+CLOUDFRONT_DOMAIN_NAME = "#{site[:subdomain]}.#{BASE_DOMAIN_NAME}"
+%>
```

Comment:
I don't think this is a good approach - it's mixing up the layers here and we should prefer cloudformation template parameters over erb wherever possible. I'll make a separate comment explaining what I would recommend instead.

Reply:
I can swap this out in the marketing stack, however on the Cloudfront Cert stack, I was intending to do one stack that creates all TLS Certs for all marketing clusters. If we were to use this parameterized model, we would need one TLS Cert stack per marketing cluster. (1:1 relationship) since the base domain name and subdomain name would need to be inputted at runtime essentially partitions the stack by subdomain and base domain name.

Reply:
A requirement that I would like to enforce is:

> Use the same cloudformation template in every environment, only use Cloudformation Parameters to make environments unique

The approach you have here violates that, because the templates themselves are going to be different, depending on what "stage" you selected when rendering the template. I'm taking a look to see if I can make a specific recommendation, but I think it might be easier to look at after this merges.

In short, I think we should render a "properties.json" file for each stack we want to deploy and use that in the cloudformation deploy step.

Reply:
To keep this PR small, could we have the `properties.json` mechanism be a follow-up PR? This PR would replace the current dev stack and be manually deployed via `aws cloudformation deploy` and then we can follow up with the properties model

Reply:
yes, making that change in a followup PR will be easier.